index.js

```diff
@@ -1,14 +1,27 @@
-const { curryN } = require('crocks')
+const { curryN }         = require('crocks')
+const { flip, is, when } = require('ramda')
+
+const addReqData = (err, req) =>
+  when(is(Error), flip(Object.assign)({
```

Comment:
I'm not sure this `when(is(Error))` is needed, since only real errors will be passed to `addReqData`.  The type-check is already done elsewhere.  And without the `when`, we won't need to `flip`.  We can do a plain-old `Object.assign`.